Userland/Libraries/LibWeb/Painting/GradientPainting.cpp

```diff
@@ -115,7 +115,7 @@ LinearGradientData resolve_linear_gradient_data(Layout::Node const& node, CSSPix
 
     auto resolved_color_stops = resolve_color_stop_positions(
         linear_gradient.color_stop_list(), [&](auto const& length_percentage) {
-            return length_percentage.to_px(node, gradient_length_px).value() / static_cast<double>(gradient_length_px);
+            return length_percentage.to_px(node, gradient_length_px).to_double() / static_cast<double>(gradient_length_px);
```

Comment:
```suggestion
            return length_percentage.to_px(node, gradient_length_px).to_float() / static_cast<float>(gradient_length_px);
```
(This value is treated as a float in `resolve_color_stop_positions()` and double precision is not required for gradients)